apps/researcher/src/messages/nl/messages.json

```diff
@@ -153,54 +153,54 @@
   "caring": "Verzorging",
-  "caringLabel": "De Verzorgingskennisgeving wordt gebruikt om aan te geven dat gespecialiseerde culturele zorg vereist is. Zorg voor items of collecties vereist specifieke gemeenschapsrichtlijnen voor opslag en handling.",
+  "caringLabel": "De _Verzorgings-notice_ wordt gebruikt om aan te geven dat gespecialiseerde culturele zorg vereist is. Zorg voor items of collecties vereist specifieke gemeenschapsrichtlijnen voor opslag en hantering.",
   "genderAware": "Genderbewust",
-  "genderAwareLabel": "De Genderbewust Kennisgeving wordt gebruikt om aan te geven dat er specifieke genderverantwoordelijkheden zijn die van invloed zijn op wie toegang kan krijgen tot, kijken naar en/of omgaan met dit item of deze collectie.",
-  "leaveUndisturbed": "Ongestoord Laten",
-  "leaveUndisturbedLabel": "De Ongestoord Laten Kennisgeving wordt gebruikt om aan te geven dat items in een collectie niet mogen worden verstoord. Ze mogen alleen onder zeer beperkte omstandigheden volgens gemeenschapsrichtlijnen worden verplaatst of verstoord.",
+  "genderAwareLabel": "De _Genderbewust-notice_ wordt gebruikt om aan te geven dat er specifieke genderverantwoordelijkheden zijn die van invloed zijn op wie toegang kan krijgen tot, kijken naar en/of omgaan met dit item of deze collectie.",
+  "leaveUndisturbed": "Onverstoord Laten",
+  "leaveUndisturbedLabel": "De _Onverstoord-Laten-notice_ wordt gebruikt om aan te geven dat items in een collectie niet mogen worden verstoord. Ze mogen alleen onder zeer beperkte omstandigheden, volgens gemeenschapsrichtlijnen, worden verplaatst of verstoord.",
   "safety": "Veiligheid",
-  "safetyLabel": "De Veiligheidskennisgeving wordt gebruikt om aan te geven dat uiterste voorzichtigheid en zorg nodig is bij het bekijken en behandelen van dit item of deze collectie voor het welzijn en de gezondheid van individuen of gemeenschappen.",
+  "safetyLabel": "De _Veiligheids-notice_ wordt gebruikt om aan te geven dat uiterste voorzichtigheid en zorg nodig is bij het bekijken en behandelen van dit item of deze collectie, voor het welzijn en de gezondheid van individuen of gemeenschappen.",
   "viewing": "Bekijken",
-  "viewingLabel": "De Bekijkkennisgeving wordt gebruikt om aan te geven dat er beperkingen zijn op wie, hoe en wanneer een item/collectie kan worden bekeken, gedistribueerd en gecirculeerd. Dit omvat bestaande afbeeldingen of representaties van een bestaand of gedeaccesioneerd item binnen een collectie.",
+  "viewingLabel": "De _Bekijken-notice_ wordt gebruikt om aan te geven dat er beperkingen zijn op wie, hoe en wanneer een item/collectie kan worden bekeken, gedistribueerd en gecirculeerd. Dit omvat bestaande afbeeldingen of representaties van een bestaand of gedeaccesioneerd item binnen een collectie.",
```

Comment:
Wat is 'gedeaccesioneerd' ?? mooi woord maar volgens mij geen Nederlands :)

Reply:
ik maak er 'afgestoten' van , een term veel gebruikt in deze context